src/index.js

```diff
@@ -1,12 +1,14 @@
 import 'assets/styles/main.scss';
 import VideoPlayer from './video-player';
 import { assign } from 'utils/assign';
+import { pick } from './utils/object';
+import { CLOUDINARY_CONFIG_PARAM } from './video-player.const';
 
 if (window.cloudinary && window.cloudinary.Cloudinary) {
   console.warn('For version 1.9.0 and above, cloudinary-core is not needed for using the Cloudinary Video Player');
 }
 
-const getConfig = (playerOptions = {}, cloudinaryConfig) => assign(playerOptions, { cloudinaryConfig: cloudinaryConfig || playerOptions });
+const getConfig = (playerOptions = {}, cloudinaryConfig) => assign(playerOptions, { cloudinaryConfig: cloudinaryConfig || pick(playerOptions, CLOUDINARY_CONFIG_PARAM) });
```

Comment:
this is becoming quite confusing.
if we call `getConfig({cloud_name: 'foo'})`
you'll get in return:
```
{
  cloud_name: 'foo'
  cloudinaryConfig: {
    cloud_name: 'foo'
  }
}
```
correct? is that what we want?